main.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+
+
+int main(int argc, char *argv[]) {
+    // std::ofsteam configfile;
+   // configfile.open("known_recipients.txt");
+
+    //std::string argument1 = "";
+
+//std::string user_input = "";
+
+//Location
+
+//if(argument = init)
+    std::string first_and_last ="", email = "", cypher = "", timezone, path ="";
+
+    std::cout << "Enter your first and last name, please" << std::endl;
+    std::getline(std::cin, first_and_last);
+
+        while(first_and_last == ""){
+        std::cout << "Please enter your first and last name to continue" << std::endl;
```

Comment:
I am going to assume maybe you didn't push the code you are referring to but your `getline` should be before and inside your loop so the loop has a chance to change its test variable so it can exit someday